src/features/collaborator-avatars/CollaboratorAvatarItem.js

```diff
@@ -35,9 +34,15 @@ const CollaboratorAvatarItem = (props: Props) => {
 
     const avatarInstance =
         hasCustomAvatar && avatarUrl ? (
-            <Avatar avatarUrl={avatarUrl} name={name} {...rest} />
+            <Avatar
+                avatarUrl={avatarUrl}
+                name={name}
+                {...rest}
```

Comment:
Minor: I'm curious about why the `rest` props appear in the middle of the list. Are `isExternal` and `shouldShowExternal` overriding some of the `rest` props?

Reply:
oh, yeah, i suppose the convention is to put `...rest` at the end. it wasn't done intentionally; tho given the context of this component, there would be no overriding since the props used and `...rest` have the same source.